textattack/loggers/attack_log_manager.py

```diff
@@ -27,7 +35,13 @@ def enable_wandb(self):
         self.loggers.append(WeightsAndBiasesLogger())
 
     def add_output_file(self, filename):
```

Comment:
We don't need to specify color for file logging. I think the old one line code is fine.